printer_mods/arhodiewithsoul/readme.md

```diff
@@ -0,0 +1,20 @@
+Components modified or created as follows:
+
+[a]_leadscrew_block_202008_x2: Recessed T8 nut, holes line up for M3 fixings (BHCS). Heads of fixings should be slightly sub-flush to bottom surface. *Requires supports in recess - not tested without supports.
```

Comment:
We cannot merge STLs that require supports.  Either redesign the parts so support is not required (preferred), or if support is absolutely required then it must be included as part of the STL.

Reply:
Have pulled the part from the folder/ pull request for now - discussion on Discord led me to believe that the part will be redundant in the near future.